tools/depends/target/pythonmodule-pil/Makefile

```diff
@@ -33,6 +33,11 @@ PYTHONPATH=$(PILPATH):$(PYTHON_SITE_PKG)
 CFLAGS+= -target arm64-apple-darwin
 endif
 
+SED_FLAG=-i
+ifeq (darwin, $(findstring darwin, $(BUILD)))
+  SED_FLAG=-i ''
```

Comment:
don't you need to enclose it in quotes? `SED_FLAG="-i ''"`

Reply:
Short answer is no